src/components/Player.vue

```diff
@@ -641,13 +666,53 @@ li.move:not(.from) .player .overlay svg.move {
   top: 5px;
   box-shadow: 0 0 5px black;
   text-overflow: ellipsis;
-  overflow: hidden;
   padding: 0 4px;
 
   #townsquare:not(.spectator) &:hover,
   &.active {
     color: red;
   }
+
+  .pronouns {
+    display: inline-flex;
+    position: absolute;
+    max-width: 250px;
+    padding: 5px 10px;
+    left: 110%;
+    font-size: 80%;
+    background: rgba(0, 0, 0, 0.5);
+    border-radius: 10px;
+    border: 3px solid black;
+    filter: drop-shadow(0 4px 6px rgba(0, 0, 0, 0.5));
+    text-align: left;
+    justify-items: center;
+    align-content: center;
+    align-items: center;
+    pointer-events: none;
+    overflow: hidden;
```

Comment:
If you add `text-overflow: ellipsis` here, it will cut off the text in a much prettier way:
![image](https://user-images.githubusercontent.com/325521/109549635-86e5f080-7ace-11eb-9527-a3afae3cc39f.png)
It only works if you don't use `display: inline-flex` though.